src/components/Request/Table.js

```diff
@@ -216,29 +217,49 @@ export const RequestTable = ({classes, page, setPage,selected,setSelected,rowsPe
  
 
 const RequestWrapper = () => {
+  const [selected, setSelected] = React.useState([]);
+  const buttons = (status) => (
+
+   <div style={{display: 'flex', flexDirection: 'row'}}>
+                    <Fab  color="primary" aria-label="edit" style={{width: 35,height: 30, marginRight: 10,}} disabled={selected.length > 1}><EditIcon style={{width: 20, height: 20,}}  disabled={status !== "PENDING"}/></Fab>
+                    <IconButton   color="secondary" aria-label="delete" style={{width: 35,height: 35,}} disabled={selected.length > 1}><DeleteIcon style={{width: 25, height: 25,}}/></IconButton>                 
+                    <img src={SideForwardIcon} alt="slidebackIcon" onClick={() => history.push('/requests/thread')} style={{width: 35, height: 35, cursor: 'pointer', marginLeft:'7%'}} />
+                  </div>
+  )
+  const initialRows = [
+    createData(1, 'Software Engineer', 'Integrated Sytems Training', 'Rwanda, Kigali', 'Mariot Hotel', 'Pending',buttons('PENDING')),
+    // createData(ids[1], 'Software Engineer', 'Integrated Sytems Training', 'Rwanda, Kigali', 'Mariot Hotel', 'Pending',buttons,),
```

Comment:
Why still having dummy data since you integrated? I logged in but I still can see a single static request in the request table.
<img width="1371" alt="Screen Shot 2021-02-23 at 16 39 09" src="https://user-images.githubusercontent.com/54619678/108859445-e4bd9880-75f5-11eb-9526-ba459ffd6d1b.png">

Reply:
It is the same case, @ConstantineMaliza let us know if you changed something in the backend.